### ISSUE TITLE
Crash when using replacer in field mode without technic mod

### ISSUE THREAD

**Issue by OgelGames** [SwissalpS/replacer#14]
```
2020-02-20 20:45:47: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod '' in callback item_OnPlace(): ...\mods\replacer/replacer.lua:261: attempt to call field 'get_charge' (a nil value)
2020-02-20 20:45:47: ERROR[Main]: stack traceback:
2020-02-20 20:45:47: ERROR[Main]: 	...\mods\replacer/replacer.lua:261: in function <...\mods\replacer/replacer.lua:197>
```

**Comment by SwissalpS** [comment]
thanks, I'll look into that.

**Comment by SwissalpS** [comment]
[fixed in 6686a6f](https://github.com/SwissalpS/replacer/commit/6686a6f5e41738605ac7e9707eee3f23b4015025)
